apps/kyc-server/src/components/dashboard/skeletons/user-table-skeleton.tsx

```
@@ -0,0 +1,115 @@
+const HEADER_WIDTHS = [72, 88, 120, 64, 96, 112, 80, 48] as const
+const CELL_WIDTHS = [16, 16, 96, 96, 64, 80, 100, 32] as const
+
+export function UserTableSkeleton() {
+	return (
+		<div
+			className="flex w-full flex-col justify-start gap-6"
+			role="status"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)